rules/package/addon-utils.js

```diff
@@ -16,7 +16,7 @@ function rule(package) {
     return;
   }
   const addonUtilsPackageVersion = allDependencies[addonUtilsPackage];
-  if (!semver.satisfies(addonUtilsMinVersion, addonUtilsPackageVersion)) {
+  if (!semver.ltr(addonUtilsMinVersion, addonUtilsPackageVersion)) {
```

Comment:
The previous version was choking in cases where the package.json had a semver-esque range operator "^5.0.2". Hopefully this fixes it and will play friendly w/ addon-utils v6+.